src/TerritorialCouncil/Handlers/TerritorialCouncilConsularCounselorHandler.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\TerritorialCouncil\Handlers;
+
+use App\Entity\Adherent;
+use App\Entity\TerritorialCouncil\TerritorialCouncilQualityEnum;
+
+class TerritorialCouncilConsularCounselorHandler extends AbstractTerritorialCouncilHandler
+{
+    public function supports(Adherent $adherent): bool
+    {
+        return false;
```

Comment:
it will be never called?

Reply:
No, at the moment we can not manage Consular counselor.